db/migrations/20231030140000_invoice_keysend.up.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE invoices
+SET keysend = false
+WHERE keysend IS NULL;
+
+ALTER TABLE invoices
+ALTER COLUMN keysend SET DEFAULT false;
+
+ALTER TABLE invoices
+ALTER COLUMN keysend SET NOT NULL;
```

Comment:
if a default is set, do we then still need a `NOT NULL` constraint?

Reply:
Yes because I've seen the db values go from false to NULL while running integration_tests for some reason